cli/chrome-headless-render-pdf.js

```diff
@@ -32,6 +33,14 @@ if (argv['help'] || !argv['pdf'] || !argv['url']) {
 const urls = typeof argv['url'] === 'string' ? [argv['url']] : argv['url'];
 const pdfs = typeof argv['pdf'] === 'string' ? [argv['pdf']] : argv['pdf'];
 
+if (typeof argv['window-size'] === 'string' && argv['window-size'].indexOf(',') < 0) {
```

Comment:
Can you change it to `=== 2` (and change error message)?
I think that it is better to make sure that we always have exactly single comma, or maybe even better to validate (and parse) it by regexp? So we make sure that there is no letters or other characters that may cause crash.
maybe `^([0-9]+)[,x*]([0-9]+)$`, so it could also parse something like `1920x1080` or `1920*1080`